src/main/java/com/Duke/Commands/ByeCommand.java

```diff
@@ -0,0 +1,27 @@
+package com.Duke.Commands;
+
+import com.Duke.TaskManager.DukeException;
+import com.Duke.TaskManager.TaskList;
+import com.Duke.TaskManager.UI;
+
+public class ByeCommand extends Command{
+
+    private final TaskList ls;
+
+    public ByeCommand(TaskList ls) {
+        this.ls = ls;
+    }
+
+    @Override
+    public boolean isDone(){
+        return true;
+    }
+
+    public void execute(){
+        try {
+            UI.byeCalled(ls);
+        }catch (DukeException e){
```

Comment:
Please have some spacing here as well. I noticed this in other places as well perhaps you can look through your code and change them accordingly.